README.md

```diff
@@ -29,36 +29,57 @@ Duffle is a command line tool that allows you to install and manage CNAB bundles
 
 3. Build and install your first bundle (you can find the `examples` directory in this repository):
     ```console
-    $ duffle build ./examples/helloworld/
-    Step 1/5 : FROM alpine:latest
-    ---> 196d12cf6ab1
-    Step 2/5 : RUN apk add -u bash
-    ---> Using cache
-    ---> 54b3a85c5c2e
-    Step 3/5 : COPY Dockerfile /cnab/Dockerfile
-    ---> Using cache
-    ---> cd6f4ff8d83d
-    Step 4/5 : COPY app /cnab/app
-    ---> 38a482447ffd
-    Step 5/5 : CMD ["/cnab/app/run"]
-    ---> Running in 8b22055f0a37
-    ---> e5c795c2a1f4
-    Successfully built e5c795c2a1f4
-    Successfully tagged deislabs/helloworld-cnab:64dfc7c4d825fe87506dbaba6ab038eafe2a486d
-    ==> Successfully built bundle helloworld:0.1.0
+
```

Comment:
Oops, we lost the `$ duffle build ./examples/helloworld/` command here :)

Reply:
Ah! sorry I missed this! apologies.